src/udiskslinuxprovider.c

```diff
@@ -1113,7 +1121,9 @@ handle_block_uevent (UDisksLinuxProvider *provider,
     {
       handle_block_uevent_for_block (provider, action, device);
       handle_block_uevent_for_drive (provider, action, device);
+#ifdef HAVE_MDRAID
```

Comment:
I would suggest not to add any ifdefs here and only return quickly from `handle_block_uevent_for_mdraid()`.